jupyterhub_moss/templates/option_form.html

```diff
@@ -124,6 +124,7 @@
         <option value="12">12 hours</option>
       </select>
     </div>
+    {% block simple_tab_footer %}
```

Comment:
I made minimal changes for now.
The idea is to rework the tables in another PR when the parsing from slurm is separated from the display customization.